### ISSUE TITLE
Size in datatype units please

### ISSUE THREAD

**Issue by Robert-Lambert** [MicrosoftDocs/cpp-docs#1730]
Whenever documentation describes a size parameter, please include the units in which the size is expressed.  The more atomic the size unit, the better. For example, in this page, numberOfElements is described as
"Size of the destination string buffer."  Does this mean the size in 8 bit bytes? the size in TCHAR's? The size in 16 bit words?  And since you encourage the use of the _t variant, having the example in those terms would also be useful.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b9244244-b85b-c7a2-b9bc-d062bc0a3fa1
* Version Independent ID: c6f60b17-5df5-b819-6a38-5283ace473fd
* Content: [strcat_s, wcscat_s, _mbscat_s, _mbscat_s_l](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/strcat-s-wcscat-s-mbscat-s?view=vs-2019#feedback)
* Content Source: [docs/c-runtime-library/reference/strcat-s-wcscat-s-mbscat-s.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/c-runtime-library/reference/strcat-s-wcscat-s-mbscat-s.md)
* Product: **visual-cpp**
* Technology: **cpp-ucrt**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@Robert-Lambert 
Yes, the *numberOfElements* parameter description could be more explicit that it's the number of elements allocated for the *strDestination* buffer, of whichever element size is used. It's not universally the number of bytes or `wchar_t` units, because element width varies with the function type. And it's not necessarily the number of characters, because characters may occupy more than one element, in the case of MBCS characters, UTF-8, or UTF-16.

**Comment by mr-intj** [comment]
Any time I see "Elements", it means the size of whatever element the array type contains. "Count" in the name weakly implies elements (vs bytes), but the worst is when it just says "Size"...

**Comment by TylerMSFT** [comment]
We are in the process of moving away from GitHub issues for C++ documentation (we still accept pull requests). For details, please see [Announcing a new way to give feedback on Microsoft Learn](https://techcommunity.microsoft.com/t5/microsoft-learn-blog/announcing-a-new-way-to-give-feedback-on-microsoft-learn/ba-p/4027635) This issue will be closed after it is copied to an internal tracking system.
